src/main/webapp/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
```

Comment:
nit: index.html is a vague name, how about lecture-index.html

Reply:
index.html is a standard file name displayed when viewing a directory (in this case the root dir "/").  Can't be changed